spec/text_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'text'
+
+
+describe Text do
+
+  it 'sends a message' do
+    messages = double :messages, create: nil
+    account = double :account, messages: messages
+    client = double :client, account: account
+    allow(Twilio::REST::Client).to receive(:new) { client }
+    expect(messages).to receive(:create).with(body:
+      "Thank you! Your order was placed and will be delivered within the next hour",
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)